tests/Unit/RedirectsTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+namespace Automattic\LegacyRedirector\Tests\Unit;
+
+use Automattic\LegacyRedirector\Tests\Unit\MonkeyStubs;
+use WPCOM_Legacy_Redirector;
+
+/**
+ * Capability Class Unit Test
+ */
+final class RedirectsTest extends MonkeyStubs {
```

Comment:
Let's keep the focus of this test narrow for now.

```suggestion
final class NormaliseTest extends MonkeyStubs {
```

The file name will need updating as well.